### ISSUE TITLE
Donation links reappear after being hidden

### ISSUE THREAD

**Issue by daylightbrightledlight** [duplicati/duplicati#2683]
I have:
- [X] searched open and closed issues for duplicates

----------------------------------------

### Version info

**Duplicati Version:** 2.0.2.1_beta_2017-08-01
**Operating System:** macOS 10.12
**Backend:** ?

### Bug description
When pressing "click to hide" donation links, the links reappear after restarting Duplicati.

### Steps to reproduce
- Start Duplicati
- Go to Settings
- Press "Donation messages are visible, click to hide"
- Restart Duplicati or restart computer in the middle of a backup

**Actual result:** 
The donation links reappear.

**Expected result:** 
The donation links shouldn't reappear.

**Comment by kenkendk** [comment]
Can you check if you have this file:
```
~/.config/Duplicati/suppress_donation_messages.txt
```

If this file is present, no donation messages should show.

**Comment by daylightbrightledlight** [comment]
Oh. It appears as though that file was never created. Was that file supposed to be made when toggling the "Donation messages are visible, click to hide" link?

**Comment by kenkendk** [comment]
Yes, the file is supposed to be created when you click the link.

The "visit us" links are meant to help uses, so they are not shown.
The flicker is probably caused by a slight delay when loading the settings.

It sounds as if the issue is resolved?
